sdk/iot/modelsrepository/src/fetcherAbstract.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft.
+// Licensed under the MIT license.
+
+import { OperationOptions } from "@azure/core-client";
+
+export abstract class Fetcher {
+  abstract fetch(path: string, options: OperationOptions): any;
```

Comment:
It feels like fetch should at least return a promise. What does it actually return?

Reply:
fixed this.